platform/android/export/export.cpp

```diff
@@ -2536,6 +2576,22 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 				EditorNode::get_singleton()->show_warning(TTR("Building of Android project failed, check output for the error.\nAlternatively visit docs.godotengine.org for Android build documentation."));
 				return ERR_CANT_CREATE;
 			}
+
+			if (export_app_bundle == 1) {
+				List<String> copy_args;
+				String variant = p_debug ? "debug/build-debug" : "release/build-release";
+				String output_path = vformat("android/build/build/outputs/bundle/%s.aab", variant);
+
+				copy_args.push_back(ProjectSettings::get_singleton()->get_resource_path().plus_file(output_path));
+				copy_args.push_back(p_path.replace(".apk", ".aab"));
```

Comment:
Why do we replace the extension here? Is the wrong extension passed to the method?